### DISCUSSION TITLE
Crash when opening REI Config

### DISCUSSION THREAD

**Discussion by StockiesLad** [shedaniel/RoughlyEnoughItems#991] [Q & A] [answered]
[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9103637/latest.log)

**Comment by StockiesLad** [comment]
I fixed it by updating REI and all its dependencies

**Reply by Scorpions95** [reply]
> I fixed it by updating REI and all its dependencies

how can i update REI?

**Comment by StockiesLad** [comment]
Nope! the crash is happening again

**Comment by shedaniel** [comment, marked as answer]
Install Cloth Config, that is for your Minecraft version.

**Comment by StockiesLad** [comment]
Ok, thanks
